packages/@webex/plugin-meetings/src/reachability/reachability.types.ts

```diff
@@ -52,6 +73,9 @@ export type ReachabilityResultForBackend = {
   xtls: TransportResultForBackend;
 };
 
+// This is the type used when sending reachability results to the backend, with an optional isVideoMesh flag
+export type ClusterBackendResult = ReachabilityResultForBackend & {isVideoMesh?: boolean};
```

Comment:
this type is not used anywhere, also the comment is incorrect as this type is not used for sending results to the backend, backend doesn't need the `isVideoMesh` field